presto-google-sheets/src/test/java/io/prestosql/plugin/google/sheets/TestGoogleSheets.java

```diff
@@ -45,31 +55,30 @@ protected QueryRunner createQueryRunner()
             SheetsPlugin sheetsPlugin = new SheetsPlugin();
             queryRunner = DistributedQueryRunner.builder(createSession()).build();
             queryRunner.installPlugin(sheetsPlugin);
-            queryRunner.createCatalog(GOOGLE_SHEETS, GOOGLE_SHEETS, ImmutableMap.of(
-                    "credentials-path", getTestCredentialsPath(),
-                    "metadata-sheet-id", TEST_METADATA_SHEET_ID,
-                    "sheets-data-max-cache-size", "1000",
-                    "sheets-data-expire-after-write", "5m"));
+            queryRunner.createCatalog(GOOGLE_SHEETS, GOOGLE_SHEETS, new ImmutableMap.Builder<String, String>()
+                    .put("credentials-path", getTestCredentialsPath())
+                    .put("metadata-sheet-id", "foo_bar_sheet_id#Sheet1")
+                    .put("sheets-data-max-cache-size", "2000")
+                    .put("sheets-data-expire-after-write", "10m")
+                    .put("sheets-range","Sheet1")
+                    .put("sheets-value-input-option","RAW")
+                    .put("drive-permission-type","user")
```

Comment:
when a new sheet created (using create table command), how is it shared with (or visible to) any other user?